### ISSUE TITLE
Implement support for different deid strategy for each annotation type (server)

### ISSUE THREAD

**Issue by tschaffter** [nlpsandbox/phi-deidentifier#13]
- This ticket depends on #12.
- This ticket refers to the server side implementation

**Comment by cascadianblue** [comment]
I think this ticket is mostly done except for the remaining work outlined in issue #25 . Should we close this issue and just focus on the other one?

**Comment by tschaffter** [comment]
Yes, we can close this general ticket. We can open new ticket if we need to refine the deid API.
